.travis.yml

```diff
@@ -16,3 +16,6 @@ matrix:
       dist: xenial
       sudo: true
       script: tox -e py37
+    - name: "StandradJS linter for static/*.js"
+      install: npm install -g standard
+      script: standard "static/*.js" --global $ --global moment
```

Comment:
You must add --global localStorage, without it there will be errors with localStorage like:
'localStorage' is not defined.